### PR TITLE
Fix for Issues 812 prevent creation of dashboard with same name.

### PR DESCRIPTION
**Pull request by satishc1**
https://github.com/capitalone/Hygieia/issues/812

### CODE REVIEW

File: UI/src/app/dashboard/controllers/createDashboard.js [hunk 1]
```diff
@@ -92,14 +93,16 @@
 
                 dashboardData
                     .create(submitData)
-                    .then(function (data) {
-                        // TODO: error handling
```
Comment (tabladrum) [inline UI/src/app/dashboard/controllers/createDashboard.js]: Remove the TODO comment?

File: api/src/main/java/com/capitalone/dashboard/rest/DashboardController.java [hunk 2]
```diff
@@ -1,20 +1,31 @@
...
 
 import javax.validation.Valid;
 import java.util.List;
 
 import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
-import static org.springframework.web.bind.annotation.RequestMethod.*;
+import static org.springframework.web.bind.annotation.RequestMethod.DELETE;
+import static org.springframework.web.bind.annotation.RequestMethod.GET;
+import static org.springframework.web.bind.annotation.RequestMethod.POST;
+import static org.springframework.web.bind.annotation.RequestMethod.PUT;
```
Comment (tabladrum) [inline api/src/main/java/com/capitalone/dashboard/rest/DashboardController.java]: +1 for getting wild card imports out.

File: UI/src/app/dashboard/controllers/createDashboard.js (hunk 1 above)
Comment (tabladrum) [inline UI/src/app/dashboard/controllers/createDashboard.js]: Sorry I see the red color now :-)

**Comment by tabladrum** [comment]
LGTM

**Comment by tabladrum** [comment]
LGTM. :+1:

**Comment by amitmawkin** [comment]
LGTM

**Pull request merged**